siriuspy/siriuspy/pwrsupply/pructrl/prucontroller.py

```diff
@@ -339,6 +346,20 @@ def scope_read(self, device_id):
         with self._lock:
             return _dcopy(psupply.scope)
 
+    # --- IDFF parameters ---
+
+    def idff_mode_set(self, state):
+        """Change IDFF mode: True or False."""
+        self._idff_mode = state
+        if state:
+            while not self._queue.empty():  # wait until queue is empty
+                pass
```

Comment:
maybe a little sleep here?

Reply:
done

Reply:
@anacso17 , this PR is basically tested already.